test/unit/specs/components/CheckboxFilters.spec.js

```diff
@@ -1,33 +1,34 @@
 import CheckboxFilters from '../../../../src/components/filters/CheckboxFilters'
-import { shallowMount } from '@vue/test-utils'
+import { mount } from '@vue/test-utils'
 
 describe('components', () => {
   describe('CheckboxFilters', () => {
     describe('html', () => {
       let wrapper, boxes
       beforeEach(() => {
-        wrapper = shallowMount(CheckboxFilters, {
+        wrapper = mount(CheckboxFilters, {
```

Comment:
why is mount needed instead of shallow mount ?

Reply:
You can do both.
A shallow mount renders the child components as html tags.
A full mount renders the child components all the way down, so here it renders <input> elements and such.

Since the existing tests test the DOM elements, I've switched to a full mount.
Then this test checks that things keep working like they did.